adafruit_am2320.py

```diff
@@ -36,6 +36,7 @@
 
 from adafruit_bus_device.i2c_device import I2CDevice
 from micropython import const
+from busio import I2C
```

Comment:
lets add the try block with `typing` import, even though it's not used here. This will prevent the import from occupying RAM when it runs on a microcontroller.

it also will need the comment to disable pylint being made since it thinks the import will be unused.
```python
try:
    import typing  # pylint: disable=unused-import
    from busio import I2C
except ImportError:
    pass
```

Reply:
Good thought! I had considered it, but felt a little strange adding the pylint bypass. I'll add that!

Reply:
Done!